SoD_SP_Public/Next Version/SoD_SP.ini

```diff
@@ -181,40 +181,40 @@ BerserkAI = 1
 # Es zeigt die Rohstoffquellen (Gold, Holz, usw.) im 123,456X Format. X ist eine Zehnerpotenz (z.B. 123,456,789 wird sich als 123,456K zeigen)
 # Εμφανίζει τους φυσικούς πόρους (χρυσό, ξύλο κλπ.) χρησιμοποιώντας τη μορφή 123,456Χ, όπου Χ είναι μια δύναμη του δέκα (π.χ. το 123,456,789 θα εμφανίζεται ως 123,456K)
 # Pokazuje ilość surowców (złoto, drewno, itp.) używając formatowania 123,456X , gdzie X jest odpowiednim przedrostkiem liczby 10 (np. ilość 123,456,789 zostanie wyświetlona jako 123,456K).
-ResourcesFormat = 1
+ResourcesFormat = 0
 
 # Combat textbox remains open after right-clicking on an opponent (lets you see some spells duration etc.)
 # Изменить поведение информационного окна при клике правой кнопкой мыши на вражеском существе (окно остается открытым)
 # Die Kampf-Textbox bleibt geöffnet nach Rechtsklick auf einen Gegner (Es erlaubt dem Spieler die Dauer einiger Zaubersprüchen zu sehen usw.)
-# Το πλαίσιο κειμένου στη μάχη παραμένει ανοικτό μετά το δεξί κλικ σε αντίπαλο (σας επιτρέπει να δείτε τη διάρκεια που έχουν ορισμένα ξόρκια κλπ.)  
+# Το πλαίσιο κειμένου στη μάχη παραμένει ανοικτό μετά το δεξί κλικ σε αντίπαλο (σας επιτρέπει να δείτε τη διάρκεια που έχουν ορισμένα ξόρκια κλπ.)
 # W trakcie walki okna informacyjne po kliknięciu prawym klawiszem myszki są dostępne (nie znikają) także dla jednostek wroga (umożliwią np. sprawdzenie długości działania niektórych zaklęć).
-CombatRMBTextbox = 1
+CombatRMBTextbox = 0
```

Comment:
The default ini options should not be modified, just the comments explaining what they do.